torchci/pages/testing_overhead/oncall_insights.tsx

```diff
@@ -10,9 +10,14 @@ import {
   GridRenderCellParams,
   GridValueFormatterParams,
 } from "@mui/x-data-grid";
+import GenerateIndividualTestsLeaderboard from "components/metrics/panels/GenerateIndividualTestsLeaderboard";
```

Comment:
When trying to follow the link to oncall insight, i.e. https://torchci-git-palicindividualtests2-fbopensource.vercel.app/testing_overhead/oncall_insights?oncall=module:%20functorch, I get an empty page.  Is this expected? 

Just a hunch, if this works locally for you but on on the preview page on Vercel, this might be the Rockset query limit and I have encountered before.  Basically, the results from Rockset needs to be smaller than a certain threshold of 4MB.